recipes/yaha/build.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+#strictly use anaconda build environment
+export CC=${PREFIX}/bin/gcc
+export CXX=${PREFIX}/bin/g++
```

Comment:
Can you remove this. I hope it is not needed. You should use the conda installed GCC packages.